plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestCachingJdbcClient.java

```diff
@@ -891,19 +906,34 @@ public void testEverythingImplemented()
         assertAllMethodsOverridden(JdbcClient.class, CachingJdbcClient.class);
     }
 
+    private static SingleJdbcCacheStatsAssertions assertSchemaNamesCache(CachingJdbcClient client)
+    {
+        return assertCacheStats(client, CachingJdbcCache.SCHEMA_NAMES_CACHE);
+    }
+
     private static SingleJdbcCacheStatsAssertions assertTableNamesCache(CachingJdbcClient client)
     {
         return assertCacheStats(client, CachingJdbcCache.TABLE_NAMES_CACHE);
     }
 
+    private static SingleJdbcCacheStatsAssertions assertTableHandleByNameCache(CachingJdbcClient client)
+    {
+        return assertCacheStats(client, CachingJdbcCache.TABLE_HANDLES_BY_NAME_CACHE);
+    }
+
+    private static SingleJdbcCacheStatsAssertions assertTableHandleByQueryCache(CachingJdbcClient client)
+    {
+        return assertCacheStats(client, CachingJdbcCache.TABLE_HANDLES_BY_QUERY_CACHE);
+    }
+
     private static SingleJdbcCacheStatsAssertions assertColumnCacheStats(CachingJdbcClient client)
     {
         return assertCacheStats(client, CachingJdbcCache.COLUMNS_CACHE);
     }
 
     private static SingleJdbcCacheStatsAssertions assertStatisticsCacheStats(CachingJdbcClient client)
     {
-        return assertCacheStats(client, STATISTICS_CACHE);
+        return assertCacheStats(client, CachingJdbcCache.STATISTICS_CACHE);
```

Comment:
This one is kind of an odd-one here in this commit.

Reply:
Following the same approach as other methods.